### PR TITLE
rule (Write below monitored dir): Clean up and use glob matching.

### PR DESCRIPTION
**Pull request by shane-lawrence**
Signed-off-by: Shane Lawrence <shane@lawrence.dev>

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

/kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

> If contributing rules or changes to rules, please make sure to also uncomment one of the following line:

/kind rule-update

> /kind rule-create

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area build

> /area engine

/area rules

> /area tests

> /area proposals

**What this PR does / why we need it**:
This cleans up the `Write below monitored dir` rule by changing its lists and macros. The `user_ssh_directory` macro was created with a less precise condition while awaiting https://github.com/draios/sysdig/issues/1152, but the fix has been widely available for some time now.

**Which issue(s) this PR fixes**:
Clean up only, no bug fixes.

~Fixes #~

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
I hope not!

```release-note
NONE
```

### CODE REVIEW

**Review by leogr — APPROVED** [review]
:+1:

**Comment by leogr** [comment]
/hold

to let other PRs get merged, feel free to un-hold once approved by the 2 maintainers

**Review by fntlnz — APPROVED** [review]

**Pull request merged**
